### PR TITLE
Add a student project

### PR DESCRIPTION
**Pull request by Ploypiti**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Add student project + logo
- Edit link structure bc I need to add non-github link to the website & we're not ready to open for contribution (just yet).
- [Preview](https://ploypiti.github.io/gci17.fossasia.org/)

### CODE REVIEW

**Comment by nzec** [comment]
@Ploypiti Your image must be 240x240px. Please resize that. 

This is said by the Travis Log : 
`1 image(s) have greater dimensions then specified.`

**Review by nzec — APPROVED** [review]
Looks good now!

File: index.html [hunk 1]
```diff
@@ -196,7 +196,7 @@ <h4 class="uppercase mb16 animated fadeIn">Student Projects</h4>
 					<ul id="project_holder">
 							{% for project in site.data.student_projects %}
 							<li class="student_project">
-								<a href="https://github.com/{{project.github}}">
```
Comment (ParthS007) [inline index.html]: @Ploypiti  Why you removed this? Please keep it as it was before

File: _data/student_projects.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@
 #  image: projectname.png (upload the image with that name to /img/projects/)
 
 - name: Piyush Raj
-  github: 0x48piraj/FOSSASIATask_FlaskAppForm
```
Comment (ParthS007) [inline _data/student_projects.yml:10]: Why you edited other's link ?

File: _data/student_projects.yml [hunk 3]
```diff
@@ -17,6 +17,11 @@
   image: emberjs_0x48piraj.png
 
 - name: Mochammad Rafi Al Rizkan
-  github: kol4k/eKantin
```
Comment (ParthS007) [inline _data/student_projects.yml:20]: same

**Review by ParthS007 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (nzec) [reply index.html]: I think she did it because her project was not on Github.

Comment (Ploypiti) [reply index.html]: @ParthS007 I need to add external link, but `https://github.com/` limit to link to a repo only.

**Review by anshumanv — CHANGES_REQUESTED** [review]
Can you make a repo for your project on github so you don't have to change others?

**Comment by nzec** [comment]
@anshumanv Nice idea.
I would also like to contribute to your project. @Ploypiti

**Comment by Ploypiti** [comment]
@anshumanv I'd love to but the point is I'm just a co-creator and we're not ready for open contribution yet. Of course, we're going to make a repo but it takes time, so if you could approve this one it'd be awesome (PLEASE).

**Review by anshumanv — APPROVED** [review]
LGTM

File: index.html (hunk 1 above)
Comment (NaviRocker) [reply index.html]: I recommend you to add another Link rather than Github Link. So it would be fine.

**Review by NaviRocker — APPROVED** [review]
LGTM

**Comment by NaviRocker** [comment]
@Ploypiti Please resolve the Conflicts

**Review by ParthS007 — COMMENTED** [review]
@Ploypiti  Please resolve conflicts and your PR is ready to merge 👍

**Review by ParthS007 — APPROVED** [review]

File: _data/student_projects.yml [hunk 4]
```diff
@@ -17,6 +17,11 @@
   image: emberjs_0x48piraj.png
```
Comment (ParthS007) [inline _data/student_projects.yml:17]: @Ploypiti Please correct this project link also

**Review by ParthS007 — CHANGES_REQUESTED** [review]

File: _data/student_projects.yml (hunk 4 above)
Comment (Ploypiti) [reply _data/student_projects.yml:17]: Fixed.

**Review by anshumanv — APPROVED** [review]
LGTM

**Review by ParthS007 — APPROVED** [review]

**Comment by ParthS007** [comment]
@Ploypiti  Please squash commits from next time 👍

**Pull request merged**